skfda/exploratory/visualization/_baseplot.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         fig: Optional[Figure] = None,
         axes: Union[Axes, Sequence[Axes], None] = None,
     ) -> None:
-        self.artists: np.ndarray
```

Comment:
Why this change? The type is ndarray.

Reply:
the problem is that i have to use np.array as it allows me to create an empty array (which is needed everytime i do plot)
besides i only need integers and ndarray gives me floats

Reply:
ok i just realized its type ndarray
my bad, going to correct it :)